docs/UserManuals/ConfigUI/Jenkins.md

```diff
@@ -59,6 +59,14 @@ If you're using Jenkins builds to conduct `deployments`, please select "Detect D
 - Deployment: A Jenkins build with a name that matches the given regEx will be considered as a deployment.
 - Production: A Jenkins build with a name that matches the given regEx will be considered a build in the production environment.
 
+There're two cases. If a Jenkins build has no stage, it's converted to both a cicd_task and a cicd_pipeline,
```

Comment:
This is how it works behind the scene:

- If a Jenkins job has stages, it's converted to a cicd_pipeline and its stages are converted to cicd_tasks in DevLake's domain layer schema.
- If a Jenkins job has no stage, it's converted to both a cicd_pipeline and a cicd_task whose names are the job's name.

After the conversion, the two regexes are applied to the records in the cicd_tasks table.

Reply:
fixed

Reply:
`If a Jenkins build has no stage, it's converted to both a cicd_pipeline and a cicd_task whose names are the job's name.

Cause jenkins job is converted to cicd_scope